data/tabular/sr_are_tox21/transform.py

```diff
@@ -69,6 +70,7 @@ def get_and_transform_data():
                     {"adjective": "toxic in the SR-ARE assay"},
```

Comment:
Is "SR-ARE" really "SR-Antioxidant response element toxicity"? Couldn't find anything with a quick web search.

Reply:
![image](https://github.com/OpenBioML/chemnlp/assets/103003503/0f17e396-3a54-4067-9d29-173780c8bb21)